server/routerlicious/packages/lambdas/src/utils/messageGenerator.ts

```diff
@@ -67,7 +67,7 @@ export const createRoomLeaveMessage = (clientId: string): ISignalMessage => ({
 
 /**
  * Mirrors ISignalEnvelope from runtime definitions, for signals that come from an external
- * caller (not sent by a client (so no 'clientSignalSequenceNumber') and are always addressed
+ * caller (not sent by a client (so no 'clientBroadcastSignalSequenceNumber') and are always addressed
  * to the Container (so no 'address').
  * @internal
```

Comment:
Oh man. Duplicate types like this are asking for trouble. I'd have never known about this if not for clientSignalSequenceNumber change and you finding it.
Can you add a comment in ISignalEnvelope that notes this things existence?